<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="signupButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注    册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>登    录</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>密  码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>qiu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdatePwd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
